File: CareConnect-client/careConnect-client/src/app/models/physicianScheduleResult.ts
export interface PhysicianScheduleResult {
  physicianScheduleId: number;
  physicianId: number;
  dayOfWeek: number;
  startTime: string;
  endTime: string;
  slotDurationMinutes: number;
  effectiveStartDate: Date;
  effectiveEndDate: Date | null;
  isActive: boolean;
  createdAt: Date | null;
  updatedAt: Date | null;
}

File: CareConnect/Models/Database/results/PhysicianScheduleResult.cs
namespace CareConnect.Models.Database.results
{
    public class PhysicianScheduleResult
    {
        public int PhysicianScheduleId { get; set; }
        public int PhysicianId { get; set; }
        public int DayOfWeek { get; set; }
        public TimeSpan StartTime { get; set; }
        public TimeSpan EndTime { get; set; }
        public short SlotDurationMinutes { get; set; }
        public DateTime EffectiveStartDate { get; set; }
        public DateTime? EffectiveEndDate { get; set; }
        public bool IsActive { get; set; }
        public DateTime? CreatedAt { get; set; }
        public DateTime? UpdatedAt { get; set; }
    }
}